### DISCUSSION TITLE
Web UI Version of Deep Live Cam

### DISCUSSION THREAD

**Discussion by sensahin** [hacksider/Deep-Live-Cam#317] [Show and tell]
I've forked this app to a Web UI version, allowing for real-time face swapping and video deepfake creation through a browser interface. 

This version uses Flask and Nginx, supporting live webcam streaming with options for different execution providers (CPU, CUDA, etc.). It's a straightforward web-based alternative to the original desktop application.

[Repo](https://github.com/sensahin/Deep-Live-Cam-WebUI)

**Comment by hacksider** [comment]
Awesome 😊

**Comment by zeyupan888** [comment]
```
(venv) ➜  Deep-Live-Cam-WebUI git:(main) python3 app.py


 * Serving Flask app 'app'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5000
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 136-784-207
```

nice, it's working
